src/main/java/wolox/training/controllers/ProfileController.java

```diff
@@ -0,0 +1,31 @@
+package wolox.training.controllers;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import wolox.training.exceptions.UserNotFoundException;
+import wolox.training.models.User;
+import wolox.training.repositories.UserRepository;
+
+@RestController
+@RequestMapping("/api/profile")
+@Api
+public class ProfileController {
```

Comment:
I wouldn't use a separate controller to manage the User's profile. I'd expose an endpoint in the UserController itself, something like `/users/me`, because you're not handling any objects other than the User, seems like a little bit of an overdesign 🙂